README.md

```diff
@@ -176,37 +176,34 @@ You should have [Docker](https://docker.io) installed.
 
 ### Docker image from Docker hub (option 1)
 
-We have a Docker Hub repository where we host the latest stable image of the executable file of the project. This is the easiest and fastest way to run the Substrate node with Assemblyscript Runtime.
+We have a Docker Hub repository where we host the latest stable image of the executable file of the project. This is the easiest and fastest way to run the Substrate node with Assemblyscript Runtime. To run the container, you need to provide raw chain specs file with genesis configuration.
 
 First, pull the image from the Docker Hub. 
 ``` 
 docker pull limechain/as-substrate:stable
 ```
 
-Then run the executable:
+Assuming you have the generated raw chain specs file in the current directory, run the executable:
```

Comment:
Actually, now we do not have to assume that the raw chain spec file is created in the current directory. In the example below there is {path-to-custom-spec-raw}